### ISSUE TITLE
cm-super for matplotlib latex plots

### ISSUE THREAD

**Issue by mpmdean** [jupyter/docker-stacks#1160]
I think scipy-notebook might be missing the cm-super package required for latex plots?

**What docker image you are using?**

`jupyter/scipy-notebook`

**What complete docker command do you run to launch the container (omitting sensitive values)?**
```
docker pull jupyter/scipy-notebook
jupyter-repo2docker --editable --image-name=jupyter/scipy-notebook .
```

**What steps do you take once the container is running to reproduce the issue?**

Example:

1. Switch to jupyterlab appending /lab to brower URL
2. Open notebook
3. Execute
```
import numpy as np
import matplotlib
matplotlib.rcParams['pgf.texsystem'] = 'pdflatex'
matplotlib.rcParams.update({'font.family': 'serif', 'font.size': 18,
    'axes.labelsize': 20,'axes.titlesize': 24, 'figure.titlesize' : 28})
matplotlib.rcParams['text.usetex'] = True
import matplotlib.pyplot as plt

fig, ax = plt.subplots(1,1)
x = [1, 2]
y = [1, 2]
ax.plot(x,y, label='a label')
ax.legend(fontsize=15)
```
**What do you expect to happen?**
Plot with latex labels

**What actually happens?**

error trackback
```
---------------------------------------------------------------------------
CalledProcessError                        Traceback (most recent call last)
/opt/conda/lib/python3.8/site-packages/matplotlib/texmanager.py in _run_checked_subprocess(self, command, tex)
    303         try:
--> 304             report = subprocess.check_output(command,
    305                                              cwd=self.texcache,

/opt/conda/lib/python3.8/subprocess.py in check_output(timeout, *popenargs, **kwargs)
    410 
--> 411     return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
    412                **kwargs).stdout

/opt/conda/lib/python3.8/subprocess.py in run(input, capture_output, timeout, check, *popenargs, **kwargs)
    511         if check and retcode:
--> 512             raise CalledProcessError(retcode, process.args,
    513                                      output=stdout, stderr=stderr)

CalledProcessError: Command '['latex', '-interaction=nonstopmode', '--halt-on-error', '/home/jovyan/.cache/matplotlib/tex.cache/3bbc0f8d536770a62891b7cc8788c317.tex']' returned non-zero exit status 1.

The above exception was the direct cause of the following exception:

RuntimeError                              Traceback (most recent call last)
/opt/conda/lib/python3.8/site-packages/IPython/core/formatters.py in __call__(self, obj)
    339                 pass
    340             else:
--> 341                 return printer(obj)
    342             # Finally look for special method names
    343             method = get_real_method(obj, self.print_method)

/opt/conda/lib/python3.8/site-packages/IPython/core/pylabtools.py in <lambda>(fig)
    246 
    247     if 'png' in formats:
--> 248         png_formatter.for_type(Figure, lambda fig: print_figure(fig, 'png', **kwargs))
    249     if 'retina' in formats or 'png2x' in formats:
    250         png_formatter.for_type(Figure, lambda fig: retina_figure(fig, **kwargs))

/opt/conda/lib/python3.8/site-packages/IPython/core/pylabtools.py in print_figure(fig, fmt, bbox_inches, **kwargs)
    130         FigureCanvasBase(fig)
    131 
--> 132     fig.canvas.print_figure(bytes_io, **kw)
    133     data = bytes_io.getvalue()
    134     if fmt == 'svg':

/opt/conda/lib/python3.8/site-packages/matplotlib/backend_bases.py in print_figure(self, filename, dpi, facecolor, edgecolor, orientation, format, bbox_inches, **kwargs)
   2098                            else suppress())
   2099                     with ctx:
-> 2100                         self.figure.draw(renderer)
   2101                     bbox_artists = kwargs.pop("bbox_extra_artists", None)
   2102                     bbox_inches = self.figure.get_tightbbox(renderer,

/opt/conda/lib/python3.8/site-packages/matplotlib/artist.py in draw_wrapper(artist, renderer, *args, **kwargs)
     36                 renderer.start_filter()
     37 
---> 38             return draw(artist, renderer, *args, **kwargs)
     39         finally:
     40             if artist.get_agg_filter() is not None:

/opt/conda/lib/python3.8/site-packages/matplotlib/figure.py in draw(self, renderer)
   1733 
   1734             self.patch.draw(renderer)
-> 1735             mimage._draw_list_compositing_images(
   1736                 renderer, self, artists, self.suppressComposite)
   1737 

/opt/conda/lib/python3.8/site-packages/matplotlib/image.py in _draw_list_compositing_images(renderer, parent, artists, suppress_composite)
    135     if not_composite or not has_images:
    136         for a in artists:
--> 137             a.draw(renderer)
    138     else:
    139         # Composite any adjacent images together

/opt/conda/lib/python3.8/site-packages/matplotlib/artist.py in draw_wrapper(artist, renderer, *args, **kwargs)
     36                 renderer.start_filter()
     37 
---> 38             return draw(artist, renderer, *args, **kwargs)
     39         finally:
     40             if artist.get_agg_filter() is not None:

/opt/conda/lib/python3.8/site-packages/matplotlib/axes/_base.py in draw(self, renderer, inframe)
   2628             renderer.stop_rasterizing()
   2629 
-> 2630         mimage._draw_list_compositing_images(renderer, self, artists)
   2631 
   2632         renderer.close_group('axes')

/opt/conda/lib/python3.8/site-packages/matplotlib/image.py in _draw_list_compositing_images(renderer, parent, artists, suppress_composite)
    135     if not_composite or not has_images:
    136         for a in artists:
--> 137             a.draw(renderer)
    138     else:
    139         # Composite any adjacent images together

/opt/conda/lib/python3.8/site-packages/matplotlib/artist.py in draw_wrapper(artist, renderer, *args, **kwargs)
     36                 renderer.start_filter()
     37 
---> 38             return draw(artist, renderer, *args, **kwargs)
     39         finally:
     40             if artist.get_agg_filter() is not None:

/opt/conda/lib/python3.8/site-packages/matplotlib/axis.py in draw(self, renderer, *args, **kwargs)
   1226 
   1227         ticks_to_draw = self._update_ticks()
-> 1228         ticklabelBoxes, ticklabelBoxes2 = self._get_tick_bboxes(ticks_to_draw,
   1229                                                                 renderer)
   1230 

/opt/conda/lib/python3.8/site-packages/matplotlib/axis.py in _get_tick_bboxes(self, ticks, renderer)
   1171     def _get_tick_bboxes(self, ticks, renderer):
   1172         """Return lists of bboxes for ticks' label1's and label2's."""
-> 1173         return ([tick.label1.get_window_extent(renderer)
   1174                  for tick in ticks if tick.label1.get_visible()],
   1175                 [tick.label2.get_window_extent(renderer)

/opt/conda/lib/python3.8/site-packages/matplotlib/axis.py in <listcomp>(.0)
   1171     def _get_tick_bboxes(self, ticks, renderer):
   1172         """Return lists of bboxes for ticks' label1's and label2's."""
-> 1173         return ([tick.label1.get_window_extent(renderer)
   1174                  for tick in ticks if tick.label1.get_visible()],
   1175                 [tick.label2.get_window_extent(renderer)

/opt/conda/lib/python3.8/site-packages/matplotlib/text.py in get_window_extent(self, renderer, dpi)
    903             raise RuntimeError('Cannot get window extent w/o renderer')
    904 
--> 905         bbox, info, descent = self._get_layout(self._renderer)
    906         x, y = self.get_unitless_position()
    907         x, y = self.get_transform().transform((x, y))

/opt/conda/lib/python3.8/site-packages/matplotlib/text.py in _get_layout(self, renderer)
    289 
    290         # Full vertical extent of font, including ascenders and descenders:
--> 291         _, lp_h, lp_d = renderer.get_text_width_height_descent(
    292             "lp", self._fontproperties,
    293             ismath="TeX" if self.get_usetex() else False)

/opt/conda/lib/python3.8/site-packages/matplotlib/backends/backend_agg.py in get_text_width_height_descent(self, s, prop, ismath)
    201             texmanager = self.get_texmanager()
    202             fontsize = prop.get_size_in_points()
--> 203             w, h, d = texmanager.get_text_width_height_descent(
    204                 s, fontsize, renderer=self)
    205             return w, h, d

/opt/conda/lib/python3.8/site-packages/matplotlib/texmanager.py in get_text_width_height_descent(self, tex, fontsize, renderer)
    456         else:
    457             # use dviread.
--> 458             dvifile = self.make_dvi(tex, fontsize)
    459             with dviread.Dvi(dvifile, 72 * dpi_fraction) as dvi:
    460                 page, = dvi

/opt/conda/lib/python3.8/site-packages/matplotlib/texmanager.py in make_dvi(self, tex, fontsize)
    336             texfile = self.make_tex(tex, fontsize)
    337             with cbook._lock_path(texfile):
--> 338                 self._run_checked_subprocess(
    339                     ["latex", "-interaction=nonstopmode", "--halt-on-error",
    340                      texfile], tex)

/opt/conda/lib/python3.8/site-packages/matplotlib/texmanager.py in _run_checked_subprocess(self, command, tex)
    310                 'found'.format(command[0])) from exc
    311         except subprocess.CalledProcessError as exc:
--> 312             raise RuntimeError(
    313                 '{prog} was not able to process the following string:\n'
    314                 '{tex!r}\n\n'

RuntimeError: latex was not able to process the following string:
b'lp'

Here is the full report generated by latex:
This is pdfTeX, Version 3.14159265-2.6-1.40.20 (TeX Live 2019/Debian) (preloaded format=latex)
 restricted \write18 enabled.
entering extended mode
(/home/jovyan/.cache/matplotlib/tex.cache/3bbc0f8d536770a62891b7cc8788c317.tex
LaTeX2e <2020-02-02> patch level 2
L3 programming layer <2020-02-14>
(/usr/share/texlive/texmf-dist/tex/latex/base/article.cls
Document Class: article 2019/12/20 v1.4l Standard LaTeX document class
(/usr/share/texlive/texmf-dist/tex/latex/base/size10.clo))
(/usr/share/texlive/texmf-dist/tex/latex/type1cm/type1cm.sty)

! LaTeX Error: File `type1ec.sty' not found.

Type X to quit or <RETURN> to proceed,
or enter new name. (Default extension: sty)

Enter file name: 
! Emergency stop.
<read *> 
         
l.6 \usepackage
               {type1ec}^^M
No pages of output.
Transcript written on 3bbc0f8d536770a62891b7cc8788c317.log.
```

**Comment by mpmdean** [comment]
@parente I am not really understanding why this got closed. Is there a chance that some of the different issues got mixed up?

**Comment by romainx** [comment]
@mpmdean my bad sorry. I typed the wrong PR number in a commit that has nothing to do with this issue. So the issue was closed automatically following the merge. 

Sorry for that.

**Comment by mpmdean** [comment]
Solved via PR #1169
